.github/workflows/test-builder.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     - name: Setup Go
       uses: actions/setup-go@v2
       with:
-        go-version: '1.17.8'
+        go-version: '1.19'
```

Comment:
I think you're missing some other workflow updates to Go 1.19. While this is fine, it's now contingent that #126 is merged before this PR.

Reply:
This got caught by failed actions.  Actions are passing now, so I'm thinking I found all the pieces